src/modules/RA/DataLayer/components/tabs/TableTab/TableConfigField.js

```diff
@@ -171,7 +171,8 @@ const TableConfigField = ({ label, exportEnabled, ...rest }) => {
     <>
       <Typography variant="h5" component="h2">{translate(label)}</Typography>
 
-      {error && (<Typography color="error">{translate(error)}</Typography>)}
+      {error && error.length > 0 && error.flatMap(err =>
+        err && Object.entries(err).map(e => (<Typography color="error">{e[0]}: {translate(e[1])}</Typography>)))}
```

Comment:
Maybe it's better to read with the destructuration of `e`
```jsx
err && Object.entries(err).map(([key, value]) => (<Typography color="error">{key}: {translate(value)}</Typography>)))} 
```

Reply:
Done.